deepchem/models/losses.py

```diff
@@ -599,11 +600,12 @@ def _create_pytorch_loss(self, measure='JSD', average_loss=True):
         import torch
 
         def loss(local_enc, global_enc, batch_graph_index):
+            device = local_enc.device
             num_graphs = global_enc.shape[0]
             num_nodes = local_enc.shape[0]
 
-            pos_mask = torch.zeros((num_nodes, num_graphs))
-            neg_mask = torch.ones((num_nodes, num_graphs))
+            pos_mask = torch.zeros((num_nodes, num_graphs)).to(device)
```

Comment:
I just gave an LGTM on the other PR, so we may want to rebase this one before merge